plugins/generator-1.17.1/forge-1.17.1/particle.definition.yaml

```diff
@@ -0,0 +1,10 @@
+templates:
+  - template: particle.java.ftl
+    name: "@SRCROOT/@BASEPACKAGEPATH/particle/@NAMEParticle.java"

```

Comment:
Change the path to `.../client/particle/@NAMEParticle.java`, so the file will be generated at the right place (and it will fix the PR).

Reply:
This pathand the package value in the Java file has to be the same thing, otherwise MCreator will always causes an error.

Reply:
:heavy_check_mark: Fixed

Reply:
:heavy_check_mark: Fixed
Thanks for your catch ;)